
// -----------------------------------------------------------------------------
syntax = "proto3";

message User {
    string username = 1;
    string public_key = 2;
    enum Role {
        STOKE_KEEPER = 0;
        NURSE = 1;
        PATIENT = 2;
        ORG = 3;
        DIRECTOR = 4;
    }

    Role role = 3;
}

message UserContainer {
    repeated User entries = 1;
}